test/test_datasets_samplers.py

```diff
@@ -38,55 +37,55 @@ def get_list_of_videos(num_videos=5, sizes=None, fps=None):
         yield names
 
 
-@unittest.skipIf(not io.video._av_available(), "this test requires av")
-class Tester(unittest.TestCase):
+@pytest.mark.skipif(not io.video._av_available(), reason="this test requires av")
+class TestDatasetsSamplers:
     def test_random_clip_sampler(self):
         with get_list_of_videos(num_videos=3, sizes=[25, 25, 25]) as video_list:
             video_clips = VideoClips(video_list, 5, 5)
             sampler = RandomClipSampler(video_clips, 3)
-            self.assertEqual(len(sampler), 3 * 3)
+            assert len(sampler) == 3 * 3
             indices = torch.tensor(list(iter(sampler)))
             videos = torch.div(indices, 5, rounding_mode='floor')
             v_idxs, count = torch.unique(videos, return_counts=True)
-            assert_equal(v_idxs, torch.tensor([0, 1, 2]))
-            assert_equal(count, torch.tensor([3, 3, 3]))
+            assert v_idxs == torch.tensor([0, 1, 2])
+            assert count == torch.tensor([3, 3, 3])
 
     def test_random_clip_sampler_unequal(self):
         with get_list_of_videos(num_videos=3, sizes=[10, 25, 25]) as video_list:
             video_clips = VideoClips(video_list, 5, 5)
             sampler = RandomClipSampler(video_clips, 3)
-            self.assertEqual(len(sampler), 2 + 3 + 3)
+            assert len(sampler) == 2 + 3 + 3
             indices = list(iter(sampler))
-            self.assertIn(0, indices)
-            self.assertIn(1, indices)
+            assert 0 in indices
+            assert 1 in indices
             # remove elements of the first video, to simplify testing
             indices.remove(0)
             indices.remove(1)
             indices = torch.tensor(indices) - 2
             videos = torch.div(indices, 5, rounding_mode='floor')
             v_idxs, count = torch.unique(videos, return_counts=True)
-            assert_equal(v_idxs, torch.tensor([0, 1]))
-            assert_equal(count, torch.tensor([3, 3]))
+            assert v_idxs == torch.tensor([0, 1])
```

Comment:
Sorry for the confusion: when comparing tensors, we should still use `assert_equal`.

In this file we don't want to change the existing instances of `assert_equal`.

We should just convert `self.assertEqual` into `assert a == b` (and maybe into `assert_equal` for iterables, but I doubt there are any instances of that)

Reply:
@NicolasHug should I need to revert back all those last changes?

Reply:
yes please, the existing `assert_equal` statements (as in `master`) should be left as-is